src/analysis/exploration/surfNet.m

```diff
@@ -557,23 +704,21 @@ function printMetRxnInfo(metrxnId, fields, firstFieldWtComma, printFieldName, fi
                 if iscell(modelLocal.(fields{kF}){metrxnId})
                     % concatenate if the cell content is a cell array of strings
                     ToPrint = strjoin(modelLocal.(fields{kF}){metrxnId}, '|');
-                    %fprintf('%s', modelLocal.(fields{kF}){metrxnId});
                 else
                     % cell array of strings
                     ToPrint = modelLocal.(fields{kF}){metrxnId};
                 end
             else
                 % if it is a value, format it properly
                 ToPrint = digit2disp(modelLocal.(fields{kF})(metrxnId));
-                %fprintf('%s', digit2disp(modelLocal.(fields{kF})(metrxnId)));
             end
             fprintf('%s', ToPrint);
         end
         fprintf('\n');
     end
 
 % nested function for printing formulae
-    function printFormulae(formulaeParts, dict)
+    function printFormulae(formulaeParts, dict, arrow)
```

Comment:
Would you be able to put these internal functions as separate functions?

Reply:
@laurentheirendt , do you think mean putting them as local functions but not nested functions? Or new functions in new m-files? If the latter, I am not sure if they will be useful anywhere else? Most of the inputs are very raw data. Or is that you want to get rid of all local functions?

Reply:
I was thinking of new m-files. Then, they are indexed in the search.